sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/avro/AvroTableProviderTest.java

```diff
@@ -42,20 +42,20 @@
   @Rule public TestPipeline readPipeline = TestPipeline.create();
   @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
 
-  private static final String AVRO_FIELD_NAMES = "(name VARCHAR, age BIGINT, country VARCHAR)";
+  private static final String FIELD_NAMES = "(name VARCHAR, age BIGINT, country VARCHAR)";
```

Comment:
The Avro changes are just to keep both tests consistent since they are in principle 'the same'.